src/qt/askpassphrasedialog.cpp

```diff
@@ -119,24 +120,36 @@ void AskPassphraseDialog::accept()
         {
             if(newpass1 == newpass2)
             {
-                if(model->setWalletEncrypted(true, newpass1))
+                if (m_passphrase_out)
                 {
```

Comment:
fix formatting, brackets should be on same line

Reply:
Done